lib/zendesk_apps_support/package.rb

```diff
@@ -39,6 +39,9 @@ def validate(marketplace: true, skip_marketplace_translations: false)
         errors << Validations::Requirements.call(self)
         errors << Validations::Requests.call(self)
 
+        # warning only validators with valid manifest
```

Comment:
I'll explain. Further down in this file, there is another comment that says
```
# warning only validators
Validations::Secrets.call(self)
```

warning only validators appends to `package.warnings` instead of `errors`. Errors blocks app uploads where as warnings do not. They are printed as outputs.

Initially I wanted to put `Validations::SecureSettings.call(self)` together with `Validations::Secrets.call(self)`.

After a while I thought that I should only check for this if they have a valid manifest.

Reply:
Is there a clearer way to do this? Should I just remove the comment?

Reply:
I'd reword the comment to something like `# only adds warnings` in both places - the relevant thing to explain here is why you're not doing `errors <<`, so there's no need to mention anything about the manifest IMO

Reply:
👍 Gotcha